practice/bio/style.css

```diff
@@ -0,0 +1,29 @@
+/*     */
+
+li {
+  list-style-type: none;
+}
+
+li:before {
+  content: "\261E";
+  padding-right: 10px;
+}
+
+body {
+  background-color: slategrey;
+  color: orange;
+}
+
+blockquote {
+  font-family: arial;
+}
+
+.photo {
```

Comment:
If there's only one element of a type, no need to make a class, you can just select on `img`.